src/main.js

```diff
@@ -1,84 +1,73 @@
-// Private Methods
-function extendConfig(defaults, options) {
-    let prop, extended = {};
-
-    for (prop in defaults) {
-        if (Object.prototype.hasOwnProperty.call(defaults, prop)) {
-            extended[prop] = defaults[prop];
-        }
-    }
-
-    for (prop in options) {
-        if (Object.prototype.hasOwnProperty.call(defaults, prop)) {
-            extended[prop] = options[prop];
-        }
-    }
-
-    return extended;
-};
-
 export default class Concertina {
-    constructor(el, options) {
+    constructor(el, customOptions) {
         // Config
         this.element = el; // TODO: Get this to work with multiple elements - document.querySelectorAll('blah');
-        this.defaults = {
+        this.options = {
```

Comment:
Agree with this. Was pressed for time, and this was part of some boilerplate I found somewhere.